src/library/Uno.Themes.WinUI.Markup/Theme/ToggleButton.cs

```diff
@@ -0,0 +1,379 @@
+						public static ResourceValue<Brush> PointerOver => new("TextToggleButtonForegroundCheckedPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "TextToggleButtonForegroundCheckedPressed")]
+						public static ResourceValue<Brush> Pressed => new("TextToggleButtonForegroundCheckedPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "TextToggleButtonForegroundCheckedDisabled")]
+						public static ResourceValue<Brush> Disabled => new("TextToggleButtonForegroundCheckedDisabled", true);
+					}
+
+					public static class ForegroundIndeterminate
+					{
+						[ResourceKeyDefinition(typeof(Brush), "TextToggleButtonForegroundIndeterminate")]
+						public static ResourceValue<Brush> Default => new("TextToggleButtonForegroundIndeterminate", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "TextToggleButtonForegroundIndeterminatePointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("TextToggleButtonForegroundIndeterminatePointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "TextToggleButtonForegroundIndeterminatePressed")]
+						public static ResourceValue<Brush> Pressed => new("TextToggleButtonForegroundIndeterminatePressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "TextToggleButtonForegroundIndeterminateDisabled")]
+						public static ResourceValue<Brush> Disabled => new("TextToggleButtonForegroundIndeterminateDisabled", true);
+					}
+
+					public static class BorderBrush
+					{
+						[ResourceKeyDefinition(typeof(Brush), "TextToggleButtonBorderBrush")]
+						public static ResourceValue<Brush> Default => new("TextToggleButtonBorderBrush", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "TextToggleButtonBorderBrushPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("TextToggleButtonBorderBrushPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "TextToggleButtonBorderBrushPressed")]
+						public static ResourceValue<Brush> Pressed => new("TextToggleButtonBorderBrushPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "TextToggleButtonBorderBrushDisabled")]
+						public static ResourceValue<Brush> Disabled => new("TextToggleButtonBorderBrushDisabled", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "TextToggleButtonBorderBrushHoverOverlay")]
+						public static ResourceValue<Brush> HoverOverlay => new("TextToggleButtonBorderBrushHoverOverlay", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "TextToggleButtonBorderBrushFocusedOverlay")]
+						public static ResourceValue<Brush> FocusedOverlay => new("TextToggleButtonBorderBrushFocusedOverlay", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "TextToggleButtonBorderBrushSelectedOverlay")]
+						public static ResourceValue<Brush> SelectedOverlay => new("TextToggleButtonBorderBrushSelectedOverlay", true);
+					}
+
+					public static class BorderBrushChecked
```

Comment:
Instead of creating a separate class for `BorderBrushChecked` I think we should add them to the `BorderBrush` class as `Checked`, `CheckedPointerOver`, 'CheckedPressed`, etc.

Like we currently do for [RadioButton](https://github.com/unoplatform/Uno.Themes/blob/530204ba38264e66478eebdd2abc9ce05df3b0b9/src/library/Uno.Themes.WinUI.Markup/Theme/RadioButton.cs#L73-L82):

![image](https://github.com/unoplatform/Uno.Themes/assets/4793020/16490836-00ae-4aaf-8531-c9b7c8edbee9)

Thoughts? @agneszitte / @Xiaoy312 / @dansiegel ?